### ISSUE TITLE
[$250] Group chat – E-mail address is not displayed when add user to group chat after creation

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#45764]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.9-1
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** gocemate+a677@gmail.com
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Go to FAB> Start chat> Add to group few members> Complete the creating process
2. Navigate to group chat header> Members> Invite member> Add email address is an existing Gmail account> Complete the flow
3. Go to members and notice the newly added member

## Expected Result:
When invite new member to already created group chat - member's avatar and name should be displayed. When navigate to members list there should be also email address of the newly added member

## Actual Result:
Email address of the newly added member in already created group chat should be displayed when open members list
Also, In process of adding new member in already created group chat - members avatar and name are not displayed

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/e2864003-03b2-4d1a-a6dd-c95c98f4ba12

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~014b0b6a6c4dcf8d32</li>
        <li>Upwork Job ID: 1825777670719289609</li>
        <li>Last Price Increase: 2024-09-17</li>
    </ul>
</details>

**Comment by izarutskaya** [comment]
We think this issue might be related to the #vip-vsb

**Comment by RachCHopkins** [comment]
I can't replicate. No "hidden" contacts for me:

![image](https://github.com/user-attachments/assets/2a0e41bd-0385-4d29-ac10-08d8dff16a66)

Can we retest please?

**Comment by MelvinBot** [comment]
This has been labelled "Needs Reproduction". Follow the steps here: https://stackoverflowteams.com/c/expensify/questions/16989

**Comment by RachCHopkins** [comment]
Waiting on retest

**Comment by RachCHopkins** [comment]
Waiting on retest

**Comment by RachCHopkins** [comment]
@izarutskaya can this be retested please?

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by m-natarajan** [comment]
Tester is still reproducible. 

https://github.com/user-attachments/assets/624b66e1-0a78-415a-9cbf-0293fd23ee9e

**Comment by RachCHopkins** [comment]
@m-natarajan are you saying that only one person can reliably reproduce this problem? And that is the person who reported it in the first place?

**Comment by RachCHopkins** [comment]
Waiting on confirmation that this can be repro'd by someone other than the person who reported it.

**Comment by m-natarajan** [comment]
Other tester also able to reproduce.

https://github.com/user-attachments/assets/fc320572-7ed6-4ba2-9b56-c72d4a95a486

**Comment by CyberAndrii** [comment]
Response from backend is missing the [`login`](https://github.com/Expensify/App/blob/dc1132467b7c9e975679969ab9ca9fcd3c029665/src/pages/ReportParticipantsPage.tsx#L88) property

![image](https://github.com/user-attachments/assets/b2d04944-1109-479c-8fd5-cc2301d2c40c)

**Comment by RachCHopkins** [comment]
@akinwale is there anywhere we can go with this yet?

**Comment by akinwale** [comment]
> @akinwale is there anywhere we can go with this yet?

Based on https://github.com/Expensify/App/issues/45764#issuecomment-2330199603, we can make this an internal issue.

**Comment by RachCHopkins** [comment]
Ok this is Internal now - @akinwale do you want me to unassign you?

**Comment by RachCHopkins** [comment]
FYI I have closed the Upwork job.

**Comment by RachCHopkins** [comment]
Internal, not overdue.

**Comment by RachCHopkins** [comment]
Added to [Daily Update](https://expensify.slack.com/archives/C06ML6X0W9L/p1727123721011119) earlier today, looking for an engineer.

**Comment by RachCHopkins** [comment]
I did not mean to close that.

**Comment by RachCHopkins** [comment]
No takers yet

**Comment by RachCHopkins** [comment]
Posted in [Slack](https://expensify.slack.com/archives/C06ML6X0W9L/p1728884472038299) independently of weekly update.

**Comment by RachCHopkins** [comment]
Medium priority. Engineers are focussed on High priority for Internal issues.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Firth week)

**Comment by RachCHopkins** [comment]
Thanks @mvtglobally ! I think I'll close this if it's both not a priority and we cant' repro.
